exporter/datadogexporter/internal/utils/tags_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFormatKeyValueTag(t *testing.T) {
+	tests := []struct {
+		key         string
+		value       string
+		expectedTag string
+	}{
+		{"a.test.tag", "a.test.value", fmt.Sprintf("%s:%s", "a.test.tag", "a.test.value")},
+		{"a.test.tag", "", fmt.Sprintf("%s:%s", "a.test.tag", "n/a")},
```

Comment:
nit: I would just put the explicit `"a.test.tag:a.test.value"`, otherwise we are (almost) repeating the function code